### PR TITLE
Updated Header UI

### PR DESCRIPTION
**Pull request by Birdie13-stack**
## Assigned Issue

- [x] I am assigned to the related issue below.
Screenshot 2023-10-22 at 00 40 22
Screenshot 2023-10-22 at 00 40 32
Closes #123

## Description

This PR adds a background color and some minor changes to make the header look better

## Type of PR

For example:
- [x] ✨ Feature Update

### CODE REVIEW

**Comment by ghost** [comment]
Hi @Birdie13-stack ! Can you please add a screenshot to your PR? Thanks!

File: src/App.css [hunk 1]
```diff
@@ -5,6 +5,12 @@
 
 /** HTML ELEMENTS **/
 
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
```
Comment (ghost) [inline src/App.css:13]: nice! Thanks :)

**Comment by Birdie13-stack** [comment]
> Hi @Birdie13-stack ! Can you please add a screenshot to your PR? Thanks!

So sorry, I forgot

**Comment by ghost** [comment]
No need to apologize, thanks again for contributing @Birdie13-stack !!

**Review by ghost — APPROVED** [review]

**Pull request merged**